{
    "GSD": {
        "alias": "CVE-2021-44568",
        "description": "Two heap-overflow vulnerabilities exist in openSUSE/libsolv libsolv through 13 Dec 2020 in the decisionmap variable via the resolve_dependencies function at src/solver.c (line 1940 & line 1995), which could cause a remote Denial of Service.",
        "id": "GSD-2021-44568"
    },
    "namespaces": {
        "cve.org": {
            "CVE_data_meta": {
                "ASSIGNER": "cve@mitre.org",
                "ID": "CVE-2021-44568",
                "STATE": "PUBLIC"
            },
            "affects": {
                "vendor": {
                    "vendor_data": [
                        {
                            "product": {
                                "product_data": [
                                    {
                                        "product_name": "n/a",
                                        "version": {
                                            "version_data": [
                                                {
                                                    "version_value": "n/a"
                                                }
                                            ]
                                        }
                                    }
                                ]
                            },
                            "vendor_name": "n/a"
                        }
                    ]
                }
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "eng",
                        "value": "Two heap-overflow vulnerabilities exist in openSUSE/libsolv libsolv through 13 Dec 2020 in the decisionmap variable via the resolve_dependencies function at src/solver.c (line 1940 & line 1995), which could cause a remote Denial of Service."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "eng",
                                "value": "n/a"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://github.com/openSUSE/libsolv/issues/425",
                        "refsource": "MISC",
                        "url": "https://github.com/openSUSE/libsolv/issues/425"
                    },
                    {
                        "name": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1940",
                        "refsource": "MISC",
                        "url": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1940"
                    },
                    {
                        "name": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1995",
                        "refsource": "MISC",
                        "url": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1995"
                    }
                ]
            }
        },
        "nvd.nist.gov": {
            "configurations": {
                "CVE_data_version": "4.0",
                "nodes": []
            },
            "cve": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "ID": "CVE-2021-44568"
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "en",
                            "value": "Two heap-overflow vulnerabilities exist in openSUSE/libsolv libsolv through 13 Dec 2020 in the decisionmap variable via the resolve_dependencies function at src/solver.c (line 1940 & line 1995), which could cause a remote Denial of Service."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": []
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1940",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1940"
                        },
                        {
                            "name": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1995",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://github.com/yangjiageng/PoC/blob/master/libsolv-PoCs/resolve_dependencies-1995"
                        },
                        {
                            "name": "https://github.com/openSUSE/libsolv/issues/425",
                            "refsource": "MISC",
                            "tags": [],
                            "url": "https://github.com/openSUSE/libsolv/issues/425"
                        }
                    ]
                }
            },
            "impact": {},
            "lastModifiedDate": "2022-02-21T18:15Z",
            "publishedDate": "2022-02-21T18:15Z"
        }
    }
}